eZ/Publish/Core/Persistence/Legacy/Content/FieldValue/Converter/IntegerConverter.php

```diff
@@ -121,20 +125,20 @@ public function getIndexColumn()
      *   - {@link self::HAS_MAX_VALUE}
      *   - {@link self::HAS_MIN_VALUE}.
      *
-     * @param int|null $minValue Minimum int value, or null if not set
-     * @param int|null $maxValue Maximum int value, or null if not set
+     * @param int|null|false $minValue Minimum int value, or null if not set
+     * @param int|null|false $maxValue Maximum int value, or null if not set
```

Comment:
Yes, `false` is a possible validator value (in fact, the default for max), see https://github.com/ezsystems/ezpublish-kernel/blob/master/eZ/Publish/Core/FieldType/Validator/IntegerValueValidator.php

It should be possible to use only null, but could make logic/value checking when using this even less simple.